src/App.js

```diff
@@ -0,0 +1,12 @@
+import './App.css';
+import Calculator from './Component/Calculator';
+
+function App() {
+  return (
+    <div className="App">
+      <Calculator />
+    </div>
+  );
+}
+
```

Comment:
- [ ] App.js file needs to be class-based component, kindly update it

Reply:
Done thanks for your feedback